include/net/net_pkt.h

```diff
@@ -701,7 +703,7 @@ static inline void net_pkt_set_timestamp(struct net_pkt *pkt,
 
 static inline size_t net_pkt_get_len(struct net_pkt *pkt)
 {
-	return net_buf_frags_len(pkt->frags);
```

Comment:
BTW, should we have a way to recompute the length. So something like
```
size_t net_pkt_recompute_len(pkt)
{
   pkt->length = net-buf_frags_len(pkt->frags);
   return pkt->length;
}
```
This would be handy if we want to make sure the net_pkt contains a valid length. For example if we have a buffer chain and want to attach that to net_pkt, I do that in PPP actually.

Reply:
Then use net_pkt_append_buffer(), it will append the buf chain and update the length as well (instead of recomputing it from the start it will just add the length of the chain)